tests/components/panasonic_viera/conftest.py

```diff
@@ -93,3 +97,21 @@ def mock_remote_fixture():
         return_value=mock_remote,
     ):
         yield mock_remote
+
+
+@pytest.fixture
+async def init_integration(hass: HomeAssistant, mock_remote) -> MockConfigEntry:
```

Comment:
We prefer to type the whole signature completely when adding type annotations.
```suggestion
async def init_integration(hass: HomeAssistant, mock_remote: Mock) -> MockConfigEntry:
```

Reply:
Thanks again for your feedback. Would you recommend to add type annotations as much as possible to every signature for any new code added?

Reply:
Yes, but typing is optional for this integration still. So it's not required for this PR to be merged. We just want to stay consistent when we do add typing.

Reply:
Ok thanks. I'm coming from a C# background, so I do prefer to strongly typed anyway :) Added this suggestion and added the types to the 2 new test methods.